ibllib/tests/test_io.py

```diff
@@ -813,6 +813,22 @@ def test_assert_valid_label(self):
         with self.assertRaises(TypeError):
             video.assert_valid_label(None)
 
+    def test_get_video_frames_preload_stream(self):
+        video_path = self.url
+        frames_array = video.get_video_frames_preload(video_path, frame_numbers=range(4),
+                                                      mask=np.s_[:10, :5, :])
+        self.assertEqual(frames_array.shape, (4, 10, 5, 3))
+
+        frames_list = video.get_video_frames_preload(video_path, frame_numbers=[0, 1],
+                                                     mask=np.s_[:10, :5, :], as_list=True)
+        self.assertEqual(len(frames_list), 2)
+        self.assertIsInstance(frames_list, list)
+
+        return frames_array
```

Comment:
Is this supposed to be here?  The below two lines are unreachable

Reply:
Nope. debugging leftover. Also the whole test is gone now